_posts/2021-06-14-pruning-our-backlog.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Pruning our backlog - one issue at the time"
+date: 2021-06-14
+author: Jan Ainali
+type: blogpost
+excerpt: We have a new open process for tackling long standing issues
+categories:
+  - news
+---
+
+# Pruning our backlog - one issue at the time
+
+Two months ago in one of our regular community calls for the [Standard for Public Code](https://standard.publiccode.net), our lead codebase steward for quality, [Eric Herman](https://publiccode.net/who-we-are/team/eric-herman.html), took the lead in tackling some of our oldest issues in the backlog.
```

Comment:
What did Eric actually do? Our 'lead steward taking the lead' is weirdly repetitive.

Reply:
I'll rephrase it!